app/models/manageiq/providers/ibm_power_hmc/infra_manager.rb

```diff
@@ -106,7 +106,7 @@ def self.verify_credentials(args)
     authentication = args.dig("authentications", "default")
     userid, password = authentication&.values_at("userid", "password")
     password = ManageIQ::Password.try_decrypt(password)
-    password ||= find(args["id"]).authentication_password(authtype) if args['id']
+    password ||= find(args["id"]).authentication_password("default") if args['id']
```

Comment:
NOTE we would use a dynamic `authtype` if more types of authentications were required (e.g. multiple endpoints with different services and credentials) but [[up here]](https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/pull/45/files#diff-7c3c4c9da3d6d2f60b9c2853dd82040d585e3541aea2640df4b900dca8c29acaR106) we're only using `"default"`.